errors.go

```diff
@@ -96,3 +95,18 @@ func HandleExitCoder(err error) {
 	}
 	OsExiter(1)
 }
+
+func handleMultiError(multiErr MultiError) int {
+	code := 1
+	for _, merr := range multiErr.Errors {
+		if multiErr2, ok := merr.(MultiError); ok {
+			code = handleMultiError(multiErr2)
+		} else {
+			fmt.Fprintln(ErrWriter, merr)
+			if exitErr, ok := merr.(ExitCoder); ok {
+				code = exitErr.ExitCode()
```

Comment:
I believe this would result in the last exit code being used. I think this is fine as long as we document the behavior.